frontend/src/components/common/SimpleTable.tsx

```diff
@@ -91,6 +91,9 @@ export interface SimpleTableProps {
   page?: number;
   /** Whether to show the pagination component */
   showPagination?: boolean;
+  /** Whether to show a button for the user to refresh the table when new data is
+   * available. If false, it won't be shown. By default it's true. */
+  askForRefreshOnDataChange?: boolean;
```

Comment:
Boolean options when present are true, and when not are false. This is backwards to that and is confusing. Please change it to follow boolean attribute convention.